modules/unauthenticated-users-system-webhook.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * cicd/builds/triggering-builds-build-hooks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="unauthenticated-users-system-webhook_{context}"]
+= Adding unauthenticated users to the `system:webhook` role binding
+
+As a cluster administrator, you can add unauthenticated users to the `system:webhook` role binding in {product-title} for specific namespaces. The `system:webhook` role binding allows unauthenticated users to trigger builds from external systems such as GitHub, GitLab, and Bitbucket. Unauthenticated users do not have access to private role bindings by default. This is a change from {product-title} versions before 4.16.
+
+[IMPORTANT]
+====
+Always verify compliance with your organization's security standards when modifying unauthenticated access.
+====
+
+[NOTE]
```

Comment:
Might be unavoidable, but should try to avoid putting admonitions back-to-back (per SSG). You might be able to just put this note's text in the normal text instead of in a note. Just play with it a little and see.